helm/chart/router/templates/deployment.yaml

```diff
@@ -85,12 +89,18 @@ spec:
               protocol: TCP
           livenessProbe:
             httpGet:
-              path: {{ (index .Values.router.configuration "health-check").path | default "/health" }}
-              port: {{ splitList ":" ((index .Values.router.configuration "health-check").listen | default ":9494") | last }}
+              httpHeaders:
+                - name: Content-Type
+                  value: application/json
+              path: "/?query={__typename}"
+              port: {{ .Values.containerPorts.http }}
           readinessProbe:
             httpGet:
-              path: {{ (index .Values.router.configuration "health-check").path  | default "/health" }}
-              port: {{ (splitList ":"  ((index .Values.router.configuration "health-check").listen | default ":9494")) | last }}
+              httpHeaders:
+                - name: Content-Type
+                  value: application/json
+              path: "/?query={__typename}"
+              port: {{ .Values.containerPorts.http }}
```

Comment:
cc @garypen should we retrieve this value from `.Values.configuration.supergraph.listen` and fallback to `.Values.containerPorts.http` ?

I don't think it's very important, so this could be a followup